cmd/crc/cmd/start.go

```diff
@@ -66,6 +66,8 @@ func runStart(arguments []string) {
 	}
 	if commandResult.Status == "Running" {
 		output.Outln("The OpenShift cluster is running")
+
+		output.Outln("Note: the cluster might report an error state. This is due to disabling the following operator: 'monitoring' and is expected.")
```

Comment:
It might be better to use a `warn` o/p to make this msg stand out then the rest?

Reply:
It is not only monitoring but we also disable the machine config.

Reply:
It does not complain about `machine-config` when it comes to the degraded state, according to feedback.